src/controllers/expense.controller.js

```diff
@@ -0,0 +1,148 @@
+'use strict';
+
+const expenseService = require('../services/expense.service.js');
+const userService = require('../services/user.service.js');
+const { getFilteredExpenses } = require('../utils/getFilteredExpenses.js');
+
+const getAll = (req, res) => {
+  const {
+    userId: recivedUserId,
+    categories,
+    from,
+    to,
+  } = req.query;
+  const userId = Number(recivedUserId);
+
+  const expenses = expenseService.getAll();
+
+  const filteredExpenses = getFilteredExpenses(
+    expenses,
```

Comment:
I meant it's a bad practice to keep any logic in the controller, it should be placed in the services. So you can filter in the `expenseService.getAll`